src/clj/puppetlabs/puppetserver/jruby_request.clj

```diff
@@ -62,6 +62,10 @@
   (fn [request]
     (let [environment (get-environment-from-request request)]
       (cond
+        (and (nil? environment)
+             (= "puppet-v3-environment_modules-/*/" (get-in request [:route-info :route-id])))
+        (handler request)
```

Comment:
This is probably terrible, but I wasn't sure how to get around the endpoint wanting validation for an `environment` param without making separate wrapper functions for `/v3/environment_modules` and `/v3/environment_modules?environment=:environment`. Went about it this way initially just to make some forward progress...but I'll think about a better way to do this if there is one.